kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -0,0 +1,1180 @@
+            builder.set_interface_external_id(member, CONFIG_VERSION_KEY, str(config_version))
+
+    @staticmethod
+    def _reconcile_ip_addresses(desired, actual):
+        """Phase 6: Flush and assign IP addresses on infrastructure bridges."""
+        for device, spec in desired.ip_addresses.items():
+            bash.bash_roe('ip addr flush dev %s' % device)
+            r, o, e = bash.bash_roe('ip addr add %s dev %s' % (spec.address, device))
+            if r != 0:
+                raise Exception('failed to assign IP %s to %s: %s'
+                                % (spec.address, device, e))
+            r, o, e = bash.bash_roe('ip link set %s up' % device)
```

Ruff (0.15.6)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)